src/index.css

```diff
@@ -0,0 +1,258 @@
+@import url('https://fonts.cdnfonts.com/css/pokemon-solid');
+
+body {
+  background-image: url('./assets/pokemon.webp');
+  background-size: 100%;
+  margin: 0;
+}
+
+header {
+  position: sticky;
+  background-color: #fff;
+  top: 0;
+}
+
+nav {
+  font-family: 'Pokemon Solid', sans-serif;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  padding: 0 2rem;
+}
+
+.nav-menu-ul {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 60%;
+}
+
+.nav-menu-ul a {
+  text-decoration: none;
+  color: #ed1b24;
+  letter-spacing: 2px;
+}
+
+.logo {
+  height: 50px;
+  width: 50px;
+}
+
+main {
+  padding: 5%;
+}
+
+.buttons {
+  display: flex;
+  padding: 5% 0;
+  justify-content: center;
+  gap: 5%;
+}
+
+.buttons button {
+  min-width: 60%;
+  height: 5vh;
+  border-radius: 5px;
+}
+
+footer {
+  padding: 5%;
+}
+
+.grass {
+  background-color: #6a6;
+}
+
+span {
+  border: 1px solid #000;
+  border-radius: 0.5em;
+  padding: 2px 4px;
+}
+
+.card {
+  width: 20%;
+  display: flex;
+  flex-direction: column;
+  gap: 5px;
+  padding: 2vh 7vh;
+  border-radius: 1em;
+  border: 1px solid #eee;
+  background-color: #eee;
+}
+
+.card * {
+  margin: 0;
+  padding: 0;
+}
+
+.card-img {
+  display: flex;
+  justify-content: center;
+  width: 90%;
+  margin: 0 auto;
+}
+
+.poke-image {
+  width: 90%;
+  margin: 0 auto;
+}
+
+.hidden {
+  display: none;
+}
+
+#pokemon-list {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 60px;
+}
+
+.cardModal {
+  display: flex;
+  align-items: center;
+  margin-left: 2rem;
+  width: 50%;
+}
+
+.cardModal img {
+  width: 50%;
+}
+
+.commentsTotal {
```

Comment:
You did a great job implementing the comments section. However, as per the requirements of the project, you should stick to the wireframe provided to you by the microverse. Kindly change the layout as per the wireframe and then submit it for a re-review.
| Your design  | Wireframe design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/211137452-3bb8981e-fee4-4174-8a1b-c001d45dd459.png) | ![image](https://user-images.githubusercontent.com/33268927/211137636-e1928e12-93d5-42af-a151-9ad153d1a689.png)|